editor/post-title/index.js

```diff
@@ -83,6 +87,7 @@ class PostTitle extends Component {
 	onKeyDown( event ) {
 		if ( event.keyCode === ENTER ) {
 			event.preventDefault();
+			this.props.onEnterPress();
```

Comment:
Do you think we should handle splitting? or alternatively, limit this to pressing enter at the end of the field?

Splitting also raises the question on whether we need merging back on backspace.

Reply:
>Do you think we should handle splitting? or alternatively, limit this to pressing enter at the end of the field?

Personally I don't think these are necessary. But then again, I currently view title as a distinct field from the rest of content, where there's been some prior effort around treating it more akin to _just another block_.